src/script.js

```diff
@@ -622,17 +622,18 @@ window.onload = () => {
         console.log(msgTheme);
     }
 
-    chrome.storage.local.get("loggedUser", (userDetails) => {
-        var log = document.getElementById("log");
-        if (userDetails.loggedUser && userDetails.loggedUser.email) {
-            accessToken = userDetails.loggedUser.accessToken;
-            log.innerHTML = log.innerHTML.replace("Login", "Logout");
-            log.innerHTML = log.innerHTML.replace("login.svg", "logout.png");
-        } else {
-            log.innerHTML = log.innerHTML.replace("Logout", "Login");
-            log.innerHTML = log.innerHTML.replace("logout.png", "login.svg");
-        }
-    });
+    chrome.storage.sync.get("loggedUser",function(userDetails){
```

Comment:
As arrow functions are used elsewhere, I think this function should be an arrow function for the sake of convention.

Reply:
Made the changes